android_validation_constants.py

```diff
@@ -49,3 +55,15 @@
 MAX_CHARS_IN_CHAPTER_TITLE = 36
 MAX_CHARS_IN_CHAPTER_DESCRIPTION = 152
 MAX_CHARS_IN_MISCONCEPTION_NAME = 100
+
+# The URL that Android feedback reports are sent to.
+INCOMING_ANDROID_FEEDBACK_REPORT_URL = (
+    '/appfeedbackreporthandler/incoming_android_report')
+
+# Constants used to validate Android message request headers.
+ANDROID_API_KEY = ''
+ANDROID_APP_PACKAGE_NAME = 'org.oppia.android'
+APP_VERSION_WITH_HASH_REGEXP = re.compile(
+    constants.PLATFORM_PARAMETER_APP_VERSION_WITH_HASH_REGEXP)
+APP_VERSION_WITHOUT_HASH_REGEXP = re.compile(
+    constants.PLATFORM_PARAMETER_APP_VERSION_WITHOUT_HASH_REGEXP)
```

Comment:
This doesn't seem to be used--can it be removed?

Reply:
Done